src/LineChart.ts

```diff
@@ -66,12 +70,20 @@ module n3Charts {
         eventMgr.trigger('update', data, options);
       }, true);
 
+      scope.$watch('onDatumEnter', function() {
```

Comment:
Maybe we could avoid repeating the same chunk of code with different event name...